client/my-sites/plans/main.jsx

```diff
@@ -280,13 +318,15 @@ class Plans extends Component {
 								<div className="plans__header">
 									<DomainAndPlanPackageNavigation goBackLink={ goBackLink } step={ 2 } />
 
-									<FormattedHeader
-										brandFont
-										headerText={ translate( 'Choose the perfect plan' ) }
-										align="center"
-									/>
+									<FormattedHeader brandFont headerText={ headline } align="center" />
 
-									{ isDomainUpsell && <DomainUpsellDescription /> }
+									{ isDomainUpsell && (
+										<DomainUpsellDescription
+											planType={ isFreePlan ? 'free' : currentPlanIntervalType }
```

Comment:
Was thinking if we should use a const variable here instead of hardcoding it.

Reply:
you're right, I'll do it!